app/src/main/java/com/example/parkingsystemjava/mvp/view/DateDialogView.java

```diff
@@ -0,0 +1,12 @@
+package com.example.parkingsystemjava.mvp.view;
+
+import android.app.Fragment;
+import com.example.parkingsystemjava.mvp.contract.ParkingReservationDialogContract;
+import com.example.parkingsystemjava.mvp.view.base.FragmentView;
+
+public class DateDialogView extends FragmentView implements ParkingReservationDialogContract.ReservationActivityView {
```

Comment:
if you are not using this now, it should not be here. Please remove it

Reply:
done